setup.py

```diff
@@ -636,6 +636,12 @@ def build_mx_extension(build_ext, options):
         mxnet_mpi_lib.define_macros += [('MSHADOW_USE_CUDA', '1')]
     else:
         mxnet_mpi_lib.define_macros += [('MSHADOW_USE_CUDA', '0')]
+    
+    if os.environ.get('MXNET_USE_MKLDNN'):
+        mxnet_mpi_lib.define_macros += [('MXNET_USE_MKLDNN', os.environ.get('MXNET_USE_MKLDNN'))]
+    else:
+        mxnet_mpi_lib.define_macros += [('MXNET_USE_MKLDNN', 1)]
```

Comment:
Setting to 1 by default causes CI failure as we are using non-mkldnn mxnet package in the CI. I think we need to set it to 0 by default.